### ISSUE TITLE
Monthly Chat Agendas March (2021-03-01 and 2021-02-15) Meeting Agenda 

### ISSUE THREAD

**Issue by dune73** [coreruleset/coreruleset#2008]
This is the Agenda for the Monthly CRS Chats.

The general chat is going to happen on https://owasp.slack.com in the channel #coreruleset on Monday, March 1st, at 20:30 CET. That's the 1st Monday of the month. A separate issue chat is happening at the same location, same time on Monday, March 15th. That's the 3rd Monday of the month.

Items on the Agenda:  (see previous meetings decisions: [here](https://github.com/coreruleset/coreruleset/issues/1992#issuecomment-771133088))

### Outside development
* [Blogpost](https://pentestit.medium.com/does-your-waf-have-false-positive-228f5918e402): How to test the WAF with CRS as example
* [Blogpost](https://blog.vulners.com/2021/01/21/payload-detection-waf-challenge/): Payload detection challenge testing CRS against other offerings
* There is a new agenda template to kickstart a new meeting agenda for the monthly meetings [Link](https://github.com/coreruleset/coreruleset/wiki/Monthly-Meeting-Agenda-Template)
* @dune73 will appear in the [Infosec & OSInt show podcast](https://breachsense.io/podcast) on Friday, March 5, 2021 to talk about CRS and WAFs in general.

### PRs that have been merged since the last meeting

* #1985: Removing %{MATCHED_VAR} from logging
* #1988: Documentation: Enhancement of installation process for Nginx / IIS
* #1989: del terminating whitespace in crs-setup.conf
* #1994: feat(git): ignore docker-compose environment files in git
* #1995: feat(env): add timezone variable to compose
* #1996: Avoid FP at rule 933210 when Cookie contains / (slash)
* #1997: WordPress exclusion profile improvements
* #1998: XenForo exclusion profile improvements
* #2004: wrong version in requirements file
* #2010: 942220: fix magic number
* #2011: Drop trailing period as done in other rules
* #2012: Fix False Positive in rule 932115 -  ready to be merged it seems
* #2013: Move rule 920300 from PL2 to PL3
* #2016: 930110: make bare '..' without (back)slashes not trigger
* #2020: Request header Sec-Fetch-User (920275): Allow only booleans
* #2021: Added regression tests for 920275

## Open PRs

* #1986: Anti-virus support without need to run external programs - waiting for an extensive review
* #2000: Try to fix 933210 bypass at PL1, lowering FPs - waiting for a review
* #2014: Move 941310 from PL1 to PL2 - approach is open for a discussion
* #2022: Whitelist for YAM package manager - waiting for a review

## Open PRs marked "work in progress" / *needs action*
* #1868: Migrate PR: RE2 compatibility for 920120 - waiting for work by @franbuehler 
* #1934: feat(ci): upgrades ftw to python 3 supported version - waiting for FTW python upgrade
* #1938: V3.4/dev ftw main - work in progress
* #1958: Fix for issue #1922 - finished by @azurit, but a ModSec oddity leads to a stupid rule with information missing in the alert - or chain the rule differently and runtime parameter rule exclusions no longer work
* #1975: Nextcloud 20 false-positives - waiting for review by @azurit
* #1993: Draft: Plugins support + response body decompress plugin - waiting to be morphed into a plugin
 
## Other items

* What's up with the next release?
* Are we generally open to enter an agreement with a sponsor to provide specific rule exclusions and to maintain them over time?
* Proposed course forward for ftw:
  We are stuck on python 2 with ftw and it's no longer supported. Yet upgrading is tricky.
  * step 1: adopt ftw for py3 with minimal adjustments to tests. According to @fzipi there is no way around adopting 2 existing tests since the formatting of binary data as `raw_request` no longer works the same way. We need to use `encoded_request`.  @fzipi thinks he can do a PR within 10 days.
  * step 2: run ftw and new go-ftw (written in Go by @fzipi) version in parallel. Tests would be transformed by a script to work with go-ftw
  * step-3: replace ftw with go-ftw and adopt all changes to new, a wee bit cleaner yaml test format with stricter variable typing. go-ftw is way faster than our ftw. There is still quite some work with tests and better interfaces to make this viable.
* [CRS plugin architecture](https://github.com/coreruleset/coreruleset/issues/2003#issuecomment-773937871) - this is a fleshed out proposal with a plugin architecture and the new include statements that load it by default.

## Open Issues - Separate Issues Meeting (Monday, March 15th)

Of the 8 issues discussed in February, 4 have been closed in the meantime. The other 4 are taken up here again (in the slots 1-4).

We generally cover 10 issues per month in a separate issue meeting. Add them as you see fit.

* Issue slot 1: #1867 FP: 941120 (PL1) base64 encoded string in param - assigned to @csanders-git
* Issue slot 2: #1999 REQUEST-912-DOS-PROTECTION paranoia level 2 sets - assigned to @dune73
* Issue slot 3: #1589 Proposal to include monitoring agents exceptions in a new data file - assigned to @fzipi and @theMiddleBlue 
* Issue slot 4: #2007 942300 matching "for ? (" when it's expected to find OR? - assigned to @csanders-git 
* Issue slot 5: #1917 Use the upcoming version in 'ver' actions on our dev branch - ...
* Issue slot 6: #2027 Sec-CH-UA / Sec-CH-UA-Mobile request headers are excluded from validation - ...
* Issue slot 7: #1991 Go Test WAF bypasses - ...
* Issue slot 8: #1907 False Positive valid JSON input being caught by 942260 V3.1.0
* Issue slot 9: 
* Issue slot 10: 

## How to get to our slack and join the meeting?

If you are not yet on the OWASP Slack, here is your invite: https://owasp-slack.herokuapp.com/ .

Everybody is welcome to join our community chat.

**Comment by franbuehler** [comment]
# Decisions

## PRs


* #1986: Anti-virus support without need to run external programs - waiting for an extensive review
* #2000: Try to fix 933210 bypass at PL1, lowering FPs - waiting for a review
* #2014: Move 941310 from PL1 to PL2 - approach is open for a discussion
* #2022: Whitelist for YAM package manager - waiting for a review


## Other items

* 

## Open Issues - Separate Issues Meeting (Monday, March 15th)

* Issue slot 1: 
* Issue slot 2: 
* Issue slot 3: 
* Issue slot 4: 
* Issue slot 5: 
* Issue slot 6: 
* Issue slot 7: 
* Issue slot 8: 
* Issue slot 9: 
* Issue slot 10:
